easybuild/easyconfigs/p/parasail/parasail-2.4-foss-2018b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
+
+name = 'parasail'
+version = '2.4'
+
+homepage = 'https://github.com/jeffdaily/parasail'
+description = """parasail is a SIMD C (C99) library containing implementations
+ of the Smith-Waterman (local), Needleman-Wunsch (global), and semi-global
+ pairwise sequence alignment algorithms. """
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/jeffdaily/parasail/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['9eec7bfca011c2b311e652dbcba3475d43a7c094ca6983ffc51ee53b2cbb0343']
+
+builddependencies = [('CMake', '3.11.4')]
+
+sanity_check_paths = {
+    'files': ['bin/parasail_aligner', 'bin/parasail_stats',
+              'lib/libparasail.%s' % SHLIB_EXT, 'include/parasail.h'
+              ],
```

Comment:
@fizwit the closing `]` seems out of place?
maybe just add to previous line like this:
```
    'files': ['bin/parasail_aligner', 'bin/parasail_stats',
              'lib/libparasail.%s' % SHLIB_EXT, 'include/parasail.h'],
```